### PR TITLE
Add Script to Import PRs

### PR DESCRIPTION
**Pull request by siddarthkay**
This PR adds a script which requires 2 arguments : 
-> full PR link
-> branch url of the forked repository

This script will then import the branch from the forked repository and create a new PR with it.
This new PR will have title and descriptions copied over from the original PR.

This helps in running CI pipelines for external contributors.

status: ready

### CODE REVIEW

**Comment by siddarthkay** [comment]
update : Currently I have to test sync branch workflow, will make PR as ready once I've tested few cases.

**Comment by siddarthkay** [comment]
- I made this PR in my forked version of `status-mobile` repo : https://github.com/status-im/status-mobile/pull/16804
- This PR was created in our repo by the help of this script : https://github.com/status-im/status-mobile/pull/16805

I made a change in original PR by adding a commit and then ran the script again and the script updated the PR with the new commit.

I think this is good enough to merge.

File: scripts/import-pr.sh [hunk 1]
```diff
@@ -0,0 +1,55 @@
+#!/bin/bash
+
```
Comment (J-Son89) [inline scripts/import-pr.sh:2]: I think a nice feature will be to have the url of the original pr at the top of description so we can directly link to that 👍

Comment (siddarthkay) [reply scripts/import-pr.sh:2]: I can try that, should be easy enough to do in this PR.

File: scripts/import-pr.sh [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+# Extracting the repo URL and branch name
+FULL_URL="$1"
+REPO_URL=$(echo "$FULL_URL" | awk -F "/tree/" '{print $1}')
+REPO_NAME=$(basename "$REPO_URL")
+NEW_REMOTE_URL="https://github.com/status-im/status-mobile"
+
+# Clone the repo if not already cloned
+if [ ! -d "$REPO_NAME" ]; then
+    git clone "$REPO_URL"
```
Comment (jakubgs) [inline scripts/import-pr.sh:30]: Why are we cloning the repo? I don't get it? You know you can just checkout a PR branch directly from `origin`?
```
git fetch origin pull/$ID/head:$BRANCHNAME
git checkout $BRANCHNAME
```
https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/reviewing-changes-in-pull-requests/checking-out-pull-requests-locally

**Review by jakubgs — APPROVED** [review]
Cute, but you probably don't need to clone anything. Also might want to call `gh auth status` before you do anything to verify user is logged in.

Also, it is recommended to use curly braces around Bash variable names.

**Comment by siddarthkay** [comment]
Thanks for the review @jakubgs! I do intend to make changes to this script 👍🏻

**Comment by siddarthkay** [comment]
Closing for now, will re-open when I resume working on it.

**Pull request closed without merging**